app/javascript/availability/components/ill_link.jsx

```diff
@@ -60,15 +67,23 @@ const IllLink = ({holding}) => {
     };
 
     const label = () => {
+        if (!hasData) {
+            return "Use ILLiad to request this item";
+        }
+
         if (availability.isMicroform(holding) || availability.isArchivalThesis(holding)) {
             return "Request Scan - Penn State Users";
         }
 
         return availability.illiadLocations[holding.locationID];
     };
 
+    const linkTarget = () => {
+        return hasData ? null : "_blank";
+    };
+
     return (
-        <a href={url}>
+        <a href={url} target={linkTarget()}>
```

Comment:
@nicktiberi Wondering if  `a` href parts move to components as well?

Reply:
@banukutlu - we could definitely do that. Maybe this could just be called `<SpinnerLink />` or just `<Link />`? Any preference?

Reply:
oh yeah both names makes sense actually, the component will include with the link and the spinner. I am fine with either of your suggestions, will leave it to you :)